### PR TITLE
docs: Add examples for awscc_securityhub_hub

### PR DESCRIPTION
**Pull request by acwwat**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

### CODE REVIEW

File: docs/resources/securityhub_hub.md [hunk 1]
```diff
@@ -10,7 +9,28 @@ description: |-
 
 The AWS::SecurityHub::Hub resource represents the implementation of the AWS Security Hub service in your account. One hub resource is created for each Region in which you enable Security Hub.
 
+## Example Usage
 
+### Basic Usage
+
+```terraform
+resource "awscc_securityhub_hub" "example" {
+  tags = [{
```
Comment (quixoticmonk) [inline docs/resources/securityhub_hub.md:18]: `awscc_securityhub_hub` is one of the resources which has an issue on tags schema. Here it is a map of string compared to the attributes list we have on others. I would suggest removing the tags on this resource for the timebeing.

Comment (acwwat) [reply docs/resources/securityhub_hub.md]: @quixoticmonk Removed and updated the commit, thanks.

(P.S. I typically amend the commit and force-push to keep things cleaner. If you prefer additional commits in the future, please let me know.)

Comment (quixoticmonk) [reply docs/resources/securityhub_hub.md]: Either one works as these would be squashed before merging. Appreciate doing the changes.

**Review by quixoticmonk — APPROVED** [review]
LGTM

**Comment by marcosentino** [comment]
@acwwat Thank you for the contribution!

**Pull request merged**